addons/transform/org.openhab.transform.feed.test/src/test/resources/output/rss_1.0.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns="http://purl.org/rss/1.0/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:taxo="http://purl.org/rss/1.0/modules/taxonomy/">
+  <channel rdf:about="http://www.feedforall.com">
+    <title>Sample Feed</title>
```

Comment:
Hm, I am not 100 percent sure, any license information is missing. As you see they include some ads to feedforall.com (from where they are taken), so I don't think this should be a problem.

I don't have much experience in this field to be honest, what do you think is the best to do in this case ?